packages/babel-preset-env/src/normalize-options.js

```diff
@@ -76,7 +76,7 @@ const validBrowserslistTargets = [
 ];
 
 export const normalizePluginName = (plugin: string): string =>
-  plugin.replace(/^babel-plugin-/, "");
+  plugin.replace(/^@?babel[-/](plugin-)?/, "");
```

Comment:
This also matches things like `@babel-plugin-foo`, is it intended to keep the RegExp simple?

Reply:
Tweaked it to prevent `@babel-plugin-foo`